packages/components/docs/sass.md

```diff
@@ -12364,54 +12395,97 @@ File uploader styles
   .#{$prefix}--file-container {
     display: block;
     width: 100%;
+  }
+
+  .#{$prefix}--file-btn ~ .#{$prefix}--file-container {
     margin-top: $carbon--spacing-06;
   }
 
   .#{$prefix}--file__selected-file {
-    display: flex;
+    display: grid;
+    grid-gap: $carbon--spacing-03 $carbon--spacing-05;
+    grid-template-columns: 1fr auto;
+    grid-auto-rows: auto;
     align-items: center;
-    justify-content: space-between;
-    height: rem(40px);
-    max-width: rem(300px);
+    min-height: rem(40px);
+    max-width: rem(320px);
     margin-bottom: $carbon--spacing-03;
-    padding: 0 $carbon--spacing-03 0 $carbon--spacing-05;
+    padding: $carbon--spacing-03 $carbon--spacing-05;
     background-color: $field-01;
-    overflow: hidden;
 
     &:last-child {
       margin-bottom: 0;
     }
 
-    .#{$prefix}--inline-loading__animation,
-    .#{$prefix}--loading {
+    .#{$prefix}--form-requirement {
+      grid-column-start: 1;
+      grid-column-end: -1;
+      max-height: none;
+      margin: 0;
+    }
+
+    .#{$prefix}--inline-loading__animation {
       right: -0.25rem; // offset for loading svg container
-      width: 1.5rem;
-      height: 1.5rem;
+      width: rem(22px);
+      height: rem(22px);
+    }
+
+    .#{$prefix}--loading {
+      width: rem(21px);
+      height: rem(21px);
     }
   }
 
+  // TODO: deprecate this block
+  .#{$prefix}--file__selected-file--invalid__wrapper {
+    @include focus-outline('invalid');
+    outline-width: 1px;
+    background-color: $field-01;
+    max-width: rem(320px);
+    margin-bottom: $carbon--spacing-03;
+  }
+
   .#{$prefix}--file__selected-file--invalid {
     @include focus-outline('invalid');
-    margin-bottom: $carbon--spacing-02;
+    outline-width: 1px;
+  }
+
+  .#{$prefix}--file__selected-file--invalid .#{$prefix}--file-filename {
+    color: $disabled-02;
+  }
+
+  .#{$prefix}--file__selected-file--invalid .#{$prefix}--form-requirement {
+    border-top: 1px solid $ui-03;
+    padding-top: rem(8px);
+  }
+
+  .#{$prefix}--file__selected-file--invalid
+    .#{$prefix}--form-requirement__supplement {
+    color: $text-01;
   }
 
+  // TODO: deprecate
   .#{$prefix}--file__selected-file--invalid + .#{$prefix}--form-requirement {
+    @include type-style('caption-01');
     display: block;
-    max-height: rem(200px);
+    max-height: none;
```

Comment:
Any impact to existing markup...?

Reply:
I reverted this because it's in a block that can be deprecated